travel.md

```diff
@@ -0,0 +1,56 @@
+
+<p><b>Beware of taxi soliciting tours at the airport greeting area; they might try to scam you into paying exorbitant rates (especially so in case of non-locals)</b>. The Hyderabad Traffic Police has partnered with the Airport authorities to run a counter for prepaid taxis. This is a safe option as all taxis are registered with the police. The police also runs a <b>SHE cab service</b> now, especially for women travellers, with female drivers and safety equipment such as a GPS connected to the Police Control Room.</p>
+
+<p>One can also hire a cab from many app-based services such as Uber or Ola directly to the Venue which are readily available in Hyderabad.</p>
+
+<p>If 3-4 of you are travelling together, then it is possible to directly take a cab as it will be cheaper (Please keep in mind that while these services are substantially cheaper than the radio cabs, they will charge Rs. 200 in addition to the fare as parking charges levied on them by the airport). This option is the best while travelling to the airport, as the parking charges do not apply for departures.</p>
+
+<p>Hired cars are also available from a booth just before walking outside of the airport. This gives you the advantage of paying in advance, thereby avoiding any disagreements over price. The airport can be contacted on their (toll free for BSNL/MTNL subscribers) number 1 800 419-2008 for all services and inquiries including arrivals/departure information, facilities, transport availability, etc.</p>
+
+## By Train - Secundrabad or Hyderabad
+#### To Hyderabad:
+<p>Indian Railways has service to Hyderabad from all over India. There are three major railway stations serving the twin cities: <b>Secunderabad, Hyderabad , Kachiguda</b> and a minor station at <b>Begumpet</b>. Hyderabad Deccan Station is popularly known as <b>Nampally Station</b>. Most of the trains bound for South India and North India originate from Hyderabad,and leave via Secunderabad.</p>
+
+* [Trains to Secundrabad](https://trainspy.com/static/station/(SC)SECUNDERABAD-JN)
+* [Trains to Hyderabad - Deccan](http://trainspy.com/static/station/(HYB)HYDERABAD-DECAN)
+* [Trains to Kachiguda](http://trainspy.com/static/station/(KCG)KACHEGUDA)
+
+<p>Secundrabad and Kachiguda are almost the same distance from the venue (~40 kms). Hyderabad (Nampally) Station is ~15 kms from the venue. Using the above links, you can check which trains you can get from your region.</p>
+
+<p><b>From the stations to the venue:</b> From any station you have can take a bus to Gachibowli (ORR). Just point Google Map from your location to Gachibowli and you will get the bus numbers you need to board from the station bus stop. From Gachibowli you can take a cab or auto reach the venue.</p>
```

Comment:
Can you add a section for metro as well. From Secunderabad they need to take the Metro blue line to Hitech City. From Nampally they need to take Red line and change to blue line at Ameerpet and get down at Hi-Tech city metro stop. Here is the link to metro map https://www.ltmetro.com/metro-network-map/